Framework/PythonInterface/plugins/algorithms/LRReflectivityOutput.py

```diff
@@ -194,8 +194,10 @@ def _get_value(name, default=None):
         start_time = mtd[scaled_ws_list[0] + '_scaled'].getRun().getProperty("start_time").value
         experiment = mtd[scaled_ws_list[0] + '_scaled'].getRun().getProperty("experiment_identifier").value
         run_number = mtd[scaled_ws_list[0] + '_scaled'].getRun().getProperty("run_number").value
+        run_title = mtd[scaled_ws_list[0] + '_scaled'].getTitle()
```

Comment:
Getting the workspace `mtd[scaled_ws_list[0] + '_scaled']` into a variable would make this more readable and less error prone?
It also looks like the `readX(), readY(), readE()` would be all you need here, rather than the `dataX/Y/E()`?